src/Totals/Model.php

```diff
@@ -238,16 +200,28 @@ protected function getDescription() {
 	 * @since 2.9.0
 	 */
 	protected function getFormattedTotalRemaining() {
-		$total_remaining = ( $this->getGoal() - $this->getEarnings() ) > 0 ? ( $this->getGoal() - $this->getEarnings() ) : 0;
-		return give_currency_filter(
-			give_format_amount(
-				$total_remaining,
-				[
-					'sanitize' => false,
-					'decimal'  => false,
-				]
-			)
-		);
+		switch ( $this->metric ) {
```

Comment:
Weren't we going to reduce this to _just_ the revenue metric?

Reply:
Noted this in the PR description:
> After some more local testing, I am again hesitant to remove the metric selection feature here. Because we are not actually modifying the Give Totals shortcode functionality in any way, it feels fine to include this minor improvement over the original implementation. For this PR, I have opted to leave it in place.